js/react_native/e2e/.detoxrc.js

```diff
@@ -0,0 +1,83 @@
+/** @type {Detox.DetoxConfig} */
+module.exports = {
+  testRunner: {
+    args: {
+      '$0': 'jest',
+      config: 'e2e/jest.config.js'
+    },
+    jest: {
+      setupTimeout: 120000
+    }
+  },
+  apps: {
+    'ios.debug': {
+      type: 'ios.app',
+      binaryPath: 'ios/build/Build/Products/Debug-iphonesimulator/OnnxruntimeModuleExample.app',
+      build: 'xcodebuild -workspace ios/OnnxruntimeModuleExample.xcworkspace -scheme OnnxruntimeModuleExample -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build'
+    },
+    'ios.release': {
+      type: 'ios.app',
+      binaryPath: 'ios/build/Build/Products/Release-iphonesimulator/OnnxruntimeModuleExample.app',
+      build: 'xcodebuild -workspace ios/OnnxruntimeModuleExample.xcworkspace -scheme OnnxruntimeModuleExample -configuration Release -sdk iphonesimulator -derivedDataPath ios/build'
+    },
+    'android.debug': {
+      type: 'android.apk',
+      binaryPath: 'android/app/build/outputs/apk/debug/app-debug.apk',
+      build: 'cd android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug',
+      reversePorts: [
+        8081
+      ]
```

Comment:
why don't we have reversePorts in 'android.release'?

Reply:
The `reversePorts` feature is a ADB feature for forwarding services usually associated with physical devices. For detox tests the purpose I guess is just often to simulate user interaction with the app and verify that the app behaves correctly in response. so typically do not require such port forwarding features.

In debug mode, it might be common for developers to use reversePorts and other ADB features to debug the app with physical devices. but I guess in release mode, some of the debugging features might just got disabled and stripped out to reduce size, etc.

reverse ports section from detox doc: https://wix.github.io/Detox/blog/2022/11/10/detox-20-is-out/#reverse-ports